config/sidekiq_scheduler.yml

```diff
@@ -12,7 +12,7 @@ AppealsApi::DailyErrorReport:
   description: "Daily report of appeals errors"
 
 AppealsApi::DailyStuckRecordsReport:
-  cron: '0 0 8 * * MON-FRI America/New_York'
+  cron: '0 8 * * MON-FRI America/New_York'
```

Comment:
Fixed this cron expression while I was at it. Should run at 8:00 AM ET every Mon-Fri now.

Reply:
Weird, I thought we'd fixed that one earlier! Good catch :)